src/hooks/useAssignRolesToUserMutation.js

```diff
@@ -9,7 +9,8 @@ const useAssignRolesToUserMutation = () => {
     mutationFn: (newRole) => ky.post('roles/users', { json: newRole }).json(),
     onSuccess: async () => {
       await queryClient.invalidateQueries(namespace);
-    }
+    },
+    onError: (error) => console.error(JSON.stringify(error)) // eslint-disable-line no-console
```

Comment:
Should we be doing more than just reporting this in the console? e.g. should we actually shove this into an `alert()` for now until legit, user-friendly error handling is implemented?

Reply:
I will create a follow-on story to address this.

Reply:
Created https://folio-org.atlassian.net/browse/UIROLES-53 for this.